src/msw/srchctrl.cpp

```diff
@@ -0,0 +1,448 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        src/msw/srchctrl.cpp
+// Purpose:     wxSearchCtrl implementation - native
+// Author:      Kettab Ali
+// Created:     2019-12-23
+// Licence:     wxWindows licence
+/////////////////////////////////////////////////////////////////////////////
+
+// For compilers that support precompilation, includes "wx.h".
+#include <wx/wxprec.h>
+
+#ifdef __BORLANDC__
+    #pragma hdrstop
+#endif
+
+#if wxUSE_SEARCHCTRL
+
+#include "wx/srchctrl.h"
+
+#ifndef WX_PRECOMP
+    #include "wx/bitmap.h"
+    #include "wx/dc.h"
+    #include "wx/dcclient.h"
+    #include "wx/dcmemory.h"
+    #include "wx/log.h"
+    #include "wx/menu.h"
+    #include "wx/settings.h"
+#endif //WX_PRECOMP
+
+#include <windowsx.h> // For GET_X_LPARAM and GET_Y_LPARAM macros
```

Comment:
This should be included after `wx/msw/private.h` which includes `windows.h`, so could you please either move this line below or, maybe better, move the other one above this one.